uspace/lib/bithenge/src/failure.c

```diff
@@ -61,9 +61,9 @@
  * number BITHENGE_FAILURE_INDEX is made; a fake error is returned from this
  * call. */
 
-static int g_initialized = 0;
-static int g_failure_index = -1;
-static int g_failure_index_selected = -1;
+static errno_t g_initialized = 0;
+static errno_t g_failure_index = -1;
+static errno_t g_failure_index_selected = -1;
```

Comment:
These seem like non-errno variables to me.

Reply:
Hmm, yeah. Another file that is never built in HelenOS.

Reply:
I need to think of an easy way to list source files that haven't been compiled during the build.